docs/tutorials/developer-journey/level-2/2.2-third-party-canisters.md

```diff
@@ -0,0 +1,148 @@
+# 2.2: Using third-party canisters
```

Comment:
I would change the order of 2.2 and 2.3 since typically making inter-canister calls is a prerequisite to using third-party canisters.